src/emulator/gdbstub/src/gdb.cpp

```diff
@@ -0,0 +1,597 @@
+// Vita3K emulator project
+// Copyright (C) 2018 Vita3K team
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+#ifdef USE_GDBSTUB
+
+#include <host/state.h>
+#include <util/log.h>
+
+#include <gdbstub/functions.h>
+
+#include <cpu/functions.h>
+#include <kernel/functions.h>
+
+#include <spdlog/fmt/bundled/printf.h>
+#include <sstream>
+
+// Sockets
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <netinet/tcp.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#define LOG_GDB_EVERYTHING
+//#define LOG_GDB_DEBUG
```

Comment:
Could make this `#define LOG_GDB_DEBUG 0` and check with `#if` below, would get rid of the comment.